doc/application/application.rst

```diff
@@ -161,53 +510,102 @@ build scripts like this:
 
    target_sources(app PRIVATE src/main.c)
 
-In the above code an existing library target named 'app' is being
-configured to have the source file src/main.c. The PRIVATE keyword
-indicates that we are modifying the internals of how the library is
-being built. Whereas using the keyword 'PUBLIC' would modify how other
-libraries that link with app are built. In this case, using PUBLIC
-here would cause libraries that link with app to also include the
-source file src/main.c, behaviour that we surely do not want. The
-'PUBLIC' keyword could however be useful when modifying the include
+In the above code, an existing library target named ``app`` is
+configured to include the source file :file:`src/main.c`. The ``PRIVATE``
+keyword indicates that we are modifying the internals of how the library is
+being built. Using the keyword ``PUBLIC`` would modify how other
+libraries that link with app are built. In this case, using ``PUBLIC``
+would cause libraries that link with ``app`` to also include the
+source file :file:`src/main.c`, behaviour that we surely do not want. The
+``PUBLIC`` keyword could however be useful when modifying the include
 paths of a target library.
 
 Application CMakeLists.txt
-**************************
+==========================
+
+Every application must have a :file:`CMakeLists.txt` file. This
+file is the entry point, or top level, of the build
+system that builds the final :file:`zephyr.elf` image. :file:`zephyr.elf`
```

Comment:
Change to:

    system.  The final :file:`zephyr.elf` image contains both the application and kernel libraries.

Reply:
I think this is a follow-on adjustment to existing documentation, and will fix it as such. Please let me know that's not correct.